docs/extended_api/actions.rst

```diff
@@ -18,60 +18,70 @@ SendAPIRequest
 --------------
 
 This Action is a bit special.
-For code repetition's sake,
+For code DRYness' sake,
 all of the ``Send{METHOD}Request`` Actions
 use this Action;
 but for readability's sake,
 you will want to use
 the appropriate ``Send{METHOD}Request`` Action
 listed below.
 
-In order to show the full documentation
-for all of these Actions,
-this Action needed to be included.
-
```

Comment:
I removed this bit because i felt like it was confusing. I remember adding it because i wanted to make it clear that the `SendAPIRequest` Action wasn't the one you'd typically want to be using, even though it appears first in the documentation, but i'm sure people will be able to understand that. :P